packages/internal/src/generate/typeDefinitions.ts

```diff
@@ -214,46 +217,47 @@ export const generateTypeDefGraphQLApi = async () => {
   }
 }
 
-export const generateTypeDefGraphQLWeb = async () => {
+function generateLoadTypedefsConfig(
```

Comment:
Same problem here. Generate is the wrong verb to use here.

In RW context, "generate" is more commonly used for "create files or change the users project"

Reply:
```
export declare function loadDocuments(pointerOrPointers: UnnormalizedTypeDefPointer | UnnormalizedTypeDefPointer[], options: LoadTypedefsOptions): Promise<Source[]>;
```

New name: `getLoadDocumentsOptions`